index.bs

```diff
@@ -187,13 +191,15 @@ partial interface XRSession {
 };
 </pre>
 
-Each {{XRSession}} has an <dfn for="XRSession">environment blending mode</dfn> value, which is an enum which MUST be set to whichever of the following values best matches the behavior of imagery rendered by the session in relation to the user's surrounding environment.
+Each {{XRSession}} has an <dfn for="XRSession">environment blending mode</dfn> value, which is an enum which MUST be set to whichever of the following values best matches the behavior of imagery presented by the session to the user.
 
-  - A blend mode of <dfn enum-value for="XREnvironmentBlendMode">opaque</dfn> indicates that the user's surrounding environment is not visible at all. Alpha values in the {{XRRenderState/baseLayer}} will be ignored, with the compositor treating all alpha values as 1.0. The {{XRSession/environmentBlendMode}} MUST NOT be {{opaque}} for {{XRSessionMode/"immersive-ar"}} sessions.
+  - If the pixels of the {{XRRenderState/baseLayer}} are composited using <a>source-over</a> blending with an opaque black color, the blend mode MUST indicate of <dfn enum-value for="XREnvironmentBlendMode">opaque</dfn>. {{XRSession/environmentBlendMode}} MUST NOT be {{opaque}} for {{XRSessionMode/"immersive-ar"}} sessions.
```

Comment:
> \<a>source-over\</a>

I'm not 100% sure, but I think this, and the other places, should be wrapped in a \<dfn>\</dfn> not an \<a>\</a>?